opensearch-2 fails to build on amd64

The package opensearch-2 fails to build from source with the following error:

```
running step "git-checkout"
[git checkout] repo='https://github.com/opensearch-project/OpenSearch' dest='.' depth='1' branch='' tag='2.18.0' expcommit='99a9a81da366173b0c2b963b26ea92e15ef34547' recurse='false'
[git checkout] execute: git config --global --add safe.directory /tmp/tmp.66uPgV
[git checkout] execute: git config --global --add safe.directory /home/build
[git checkout] execute: git clone --quiet --origin=origin --config=user.name=Melange Build --config=user.email=melange-build@cgr.dev --config=advice.detachedHead=false --branch=2.18.0 --depth=1 https://github.com/opensearch-project/OpenSearch /tmp/tmp.66uPgV
[git checkout] execute: cd /tmp/tmp.66uPgV
[git checkout] tar -c . | tar -C "/home/build" -x
[git checkout] execute: cd /home/build
[git checkout] execute: git config --global --add safe.directory /home/build
[git checkout] execute: git fetch --quiet origin --depth=1 --no-tags +refs/tags/2.18.0:refs/origin/tags/2.18.0
[git checkout] execute: git checkout --quiet origin/tags/2.18.0
[git checkout] tag 2.18.0 is 99a9a81da366173b0c2b963b26ea92e15ef34547

Welcome to Gradle 8.12-20250110000000+0000!

Here are the highlights of this release:
 - Enhanced error and warning reporting with the Problems API
 - File-system watching support on Alpine Linux
 - Build and test Swift 6 libraries and apps

To honour the JVM settings for this build a single-use Daemon process will be forked. For more on this, please refer to https://docs.gradle.org/8.12-20250110000000+0000/userguide/gradle_daemon.html#sec:disabling_the_daemon in the Gradle documentation.
Daemon will be stopped at the end of the build 
Did not find udev library in operating system. Some features may not work.
Disk Store information requires libudev, which is not present.
> Task :buildSrc:reaper:compileJava
> Task :buildSrc:reaper:processResources NO-SOURCE
> Task :buildSrc:reaper:classes
> Task :buildSrc:reaper:jar
> Task :buildSrc:compileJava
/home/build/buildSrc/src/main/java/org/opensearch/gradle/test/ErrorReportingTestListener.java:185: error: <anonymous org.opensearch.gradle.test.ErrorReportingTestListener$1> is not abstract and does not override abstract method getLogTime() in TestOutputEvent
                    eventWriter.write(new TestOutputEvent() {
                                                            ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error

> Task :buildSrc:compileJava FAILED

[Incubating] Problems report is available at: file:///home/build/build/reports/problems/problems-report.html
3 actionable tasks: 3 executed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':buildSrc:compileJava'.
> Compilation failed; see the compiler output below.
  Note: Recompile with -Xlint:deprecation for details.Note: Some input files use or override a deprecated API./home/build/buildSrc/src/main/java/org/opensearch/gradle/test/ErrorReportingTestListener.java:185: error: <anonymous org.opensearch.gradle.test.ErrorReportingTestListener$1> is not abstract and does not override abstract method getLogTime() in TestOutputEvent
                      eventWriter.write(new TestOutputEvent() {
                                                              ^
  1 error

* Try:
> Check your code and dependencies to fix the compilation error(s)

BUILD FAILED in 1m 5s

The build scan was not published due to a configuration problem.

The Gradle Terms of Use have not been agreed to.

For more information, please see https://gradle.com/help/gradle-plugin-terms-of-use.

Alternatively, if you are using Develocity, specify the server location.
For more information, please see https://gradle.com/help/gradle-plugin-config.

failed to build package: unable to run package opensearch-2 pipeline: unable to run pipeline: exit status 1
```

--- a/opensearch-2.yaml
+++ b/opensearch-2.yaml
@@ -5,7 +5,7 @@
 package:
   name: opensearch-2
   version: 2.18.0
-  epoch: 0
+  epoch: 1
   description: Open source distributed and RESTful search engine.
   copyright:
     - license: Apache-2.0
@@ -25,7 +25,7 @@ environment:
       - busybox
       - ca-certificates-bundle
       - curl
-      - gradle
+      - gradle<8.12
       - openjdk-17-default-jdk
       - patch
       - unzip

